sys/fs/devfs/auto_init_devfs.c

```diff
@@ -30,8 +31,30 @@ static vfs_mount_t _devfs_auto_init_mount = {
     .mount_point = "/dev",
 };
 
+#ifdef MODULE_PERIPH_HWRNG
```

Comment:
I think we should be a bit conservative on memory and make a pseudomodule of the dev entries here, so that the user will be able to enable/disable the devfs mapping without removing the random module.

Reply:
done